.vale/styles/Silverstripe-cms/Race.yml

```diff
@@ -0,0 +1,18 @@
+---
+extends: substitution
+message: Consider using '%s' instead of '%s'.
+ignorecase: true
+level: warning
+action:
+  name: replace
+swap:
+  master copy: pass key|original
+  master key: pass key|original
+  master plan: grand scheme|guiding principles
+  master the art: become skilled
+  master: main|primary|hub|reference
```

Comment:
We've got plenty of references to master / dev-master branches including in thing like the 4.9.0-rc1 changelog, which we shouldn't touch.  I'm not sure if we actually have any master branches left (if we do, probably not to hard to just rename to main? Or just delete them. Separate card though).

Actually on a related topic are we able to exclude any changelogs that have already gone out the door, though apply this linting to changelogs going forward?  Seems quite wrong to me to be altering changelogs after the fact since they're a snapshot.

Reply:
Marking resolved since this is now out of scope.